src/library/librarycontrol.h

```diff
@@ -1,6 +1,7 @@
 #ifndef LIBRARYMIDICONTROL_H
 #define LIBRARYMIDICONTROL_H
 
+#include <memory>
```

Comment:
Hm, I think your std::make_uniques only work (make_unique a c++14 feature and we  build with std=c++11) because of transitive inclusion of util/memory.h by some path (e.g. library/library.h -> track/track.h -> track/beats.h).  Could you include util/memory.h instead (which in turn includes ```<memory>``` and declares the shim)